x/dex/keeper/msgserver/msg_server_place_orders.go

```diff
@@ -46,7 +46,16 @@ func (k msgServer) transferFunds(goCtx context.Context, msg *types.MsgPlaceOrder
 func (k msgServer) PlaceOrders(goCtx context.Context, msg *types.MsgPlaceOrders) (*types.MsgPlaceOrdersResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
-	defer ctx.ContextMemCache().IncrMetricCounter(uint32(len(msg.Orders)), sdk.ORDER_COUNT)
+	// We will report the following dex metrics:
+	//   total order count: sei_throughput_order_count
+	//   per order info: sei_throughput_order_count_[ORDER_TYPE]_[POSITION_DIRECTION]_[ORDER_STATUS]
+
+	defer func() {
+		ctx.ContextMemCache().IncrMetricCounter(uint32(len(msg.Orders)), sdk.ORDER_COUNT)
+		for _, order := range msg.Orders {
+			ctx.ContextMemCache().IncrMetricCounter(1, fmt.Sprintf("%s_%s_%s_%s", sdk.ORDER_COUNT, order.OrderType.String(), order.PositionDirection.String(), order.Status.String()))
```

Comment:
if we made order type, direction, and status labels for  `sei_throughput_order_count` it would make it easier to write alerts and dashboards